src/matroskavideo.cpp

```diff
@@ -606,11 +606,6 @@ const MatroskaTag streamRate[] = {
     ret |= static_cast<uint64_t>(buf[i]) << ((size - i - 1) * 8);
   }
 
-  if (ret < std::numeric_limits<uint64_t>::min())
-    return false;
-  if (ret > std::numeric_limits<uint64_t>::max())
-    return false;
-
```

Comment:
I think this whole function (`convertToUint64`) is just a reimplementation of `getULongLong`.

Reply:
Good Call. removed.